### PR TITLE
fix cps error in 6.3

### PR DESCRIPTION
**Pull request by luohaha**
I find that so many cps errors in exercise 6.3. So I correct them and give the right answer.

### CODE REVIEW

**Comment by luohaha** [comment]
Ask me anything, if you feel confused with my solution.

**Comment by chenyukang** [comment]
@luohaha ,
 I will review it.
Thanks very much!

File: ch6/03.scm [hunk 1]
```diff
@@ -2,144 +2,60 @@
 
 ;; 1. (lambda (x y) (p (+ 8 x) (q y)))
 ;; ==>
-(lambda (x y cont) (q y (lambda(v)
-			  (p v (cont (+ 8 x cont))))))
+(lambda (x y cont)
+  (q y (lambda (val)
+	 ((lambda (val2) (p val2 val cont))
+	  (+ 8 x)))))
```
Comment (chenyukang) [inline ch6/03.scm]: I think this should be simper as: 
```scheme
(lambda (x y cont)
  (q y (lambda(v)
         (p (+ 8 x) v cont))))
```

File: ch6/03.scm [hunk 2]
```diff
@@ -2,144 +2,60 @@
...
-
-
-(lambda (x y u v cont)
-  (g x y (f ((lambda (v1)
-               (cont (+ 1 v1 (+ u v))))))))
+  (g x y (lambda (val)
+	   ((lambda (val2)
+	      ((lambda (val3) (cont (+ 1 val3)))
+	       (f val val2)))
+	    (+ u v)))))
```
Comment (chenyukang) [inline ch6/03.scm]: more simpler?
```scheme
(lambda (x y u v cont)
  (g x y (lambda(v2)
           (f v2 (+ u v)
              (lambda (v1)
                (cont (+ 1 v1)))
              ))))
```

from the example in page 199, 
and The CPS Recipe on page 200.

File: ch6/06.scm [hunk 3]
```diff
@@ -9,30 +9,50 @@
   (+ (f (g x))
      (h (j y))))
 
-
+;; g -> j -> f -> h -> +
```
Comment (chenyukang) [inline ch6/06.scm]: Yes! 
these should be right.
seems I forgotten one level.

File: ch6/06.scm [hunk 4]
```diff
@@ -9,30 +9,50 @@
...
 (lambda (x y cont)
-  (f (g x) (lambda (v1)
-	     (+ v1 (h (j y))))))
+  (g x (lambda (val)
+	 (j y (lambda (val2)
+		(f val (lambda (val3)
+			 (h val2 (lambda (val4)
+				   (cont (+ val3 val4)))))))))))
```
Comment (chenyukang) [inline ch6/06.scm]: Yes! 
these should be right.
seems I forgotten one level.

File: ch6/03.scm (hunk 1 above)
Comment (luohaha) [inline ch6/03.scm]: Yes, you are right! I forgot '+' isn't a normal procedure call! Same as below. I will correct my PR soon.

**Comment by luohaha** [comment]
I have corrected my PR now.

**Pull request merged**